templates/search/results-contextual-fr.html

```diff
@@ -1,21 +1,35 @@
 ---
 {
-	"title": "Résultats de la recherche (Contextuel)",
+	"title": "Recherche (Contextuel)",
 	"language": "fr",
 	"altLangPage": "results-contextual-en.html",
 	"pageType": "recherche",
 	"pageclass": "page-type-search",
-	"dateModified": "2022-08-02",
+	"dateModified": "2025-07-28",
 	"share": "false",
 	"deptfeature": false
 }
 ---
 
 <div class="well">
 	<form class="mrgn-tp-sm">
-		<label class="wb-inv" for="sch-inp">Recherchez le site Web du Gouvernement du Canada</label>
-		<input class="form-control full-width" spellcheck="false" autocomplete="off" dir="ltr" id="sch-inp" name="q" type="search" data-fusion-query aria-describedby="gc-pi">
-
+		<div class="row">
+			<div class="col-md-8">
+				<label for="sch-inp" class="wb-inv">Recherche [contextuel]:</label>
+			</div>
+			<div class="col-md-4 text-right">
+				<ul class="list-inline">
+					<li><a href="/fr/sr/st.html"> Trucs de recherche</a></li>
+					<li><a href="advanced-search-fr.html">Recherche Avancée</a></li>
+				</ul>
+			</div>
+		</div>
+		<div class="input-group mrgn-tp-lg">
+			<input class="form-control full-width" spellcheck="false" autocomplete="off" dir="ltr" id="sch-inp" name="q" type="search" data-fusion-query aria-describedby="gc-pi">
+			<span class="input-group-btn">
+				<button type="submit" class="btn btn-primary"> <span class="glyphicon-search glyphicon"></span><span class="wb-inv">Recherche</span></button>
+			</span>
+		</div>
```

Comment:
For this search box, mor specifically for the submit button, please put the the magnifying glass icon without the word search to the right of the search box on the same line.